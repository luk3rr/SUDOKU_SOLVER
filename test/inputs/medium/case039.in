100730004 500900000 000040089 001004000 305010000 809003000 060080092 000600300 000090070 
