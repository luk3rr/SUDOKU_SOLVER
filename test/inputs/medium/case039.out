Solving the following grid:
1 0 0 | 7 3 0 | 0 0 4 
5 0 0 | 9 0 0 | 0 0 0 
0 0 0 | 0 4 0 | 0 8 9 
------+-------+------
0 0 1 | 0 0 4 | 0 0 0 
3 0 5 | 0 1 0 | 0 0 0 
8 0 9 | 0 0 3 | 0 0 0 
------+-------+------
0 6 0 | 0 8 0 | 0 9 2 
0 0 0 | 6 0 0 | 3 0 0 
0 0 0 | 0 9 0 | 0 7 0 


Solution found :')

1 9 6 | 7 3 8 | 2 5 4 
5 8 4 | 9 2 6 | 7 1 3 
2 3 7 | 1 4 5 | 6 8 9 
------+-------+------
6 2 1 | 8 7 4 | 9 3 5 
3 4 5 | 2 1 9 | 8 6 7 
8 7 9 | 5 6 3 | 4 2 1 
------+-------+------
7 6 3 | 4 8 1 | 5 9 2 
9 1 2 | 6 5 7 | 3 4 8 
4 5 8 | 3 9 2 | 1 7 6 

Algorithm: BFS
Total time: 282 ms
Total expanded states: 38863
